cmd/ipfs-cluster-ctl/main.go

```diff
@@ -174,7 +166,26 @@ requires authorization. implies --https, which you can disable with --force-http
 			checkErr("", errors.New("unsupported encoding"))
 		}
 
-		globalClient, err = client.NewDefaultClient(cfg)
+		var configs []*client.Config
+		var err error
+		for _, addr := range c.StringSlice("host") {
+			multiaddr, err := ma.NewMultiaddr(addr)
+			checkErr("parsing host multiaddress", err)
+
+			cfgs, err := cfg.AsTemplateForResolvedAddress(multiaddr)
+			checkErr("creating configs", err)
+			configs = append(configs, cfgs...)
+		}
+
+		for _, cfg := range configs {
+			if client.IsPeerAddress(cfg.APIAddr) && c.Bool("https") {
+				logger.Warn("Using libp2p-http. SSL flags will be ignored")
+				break;
+			}
```

Comment:
I think we can include this warning in the loop above but updating the warning message to `Using libp2p-http for %s. The https flag will be ignored for this connection`. It is ok if it appears multiple times.